readthedocs/builds/models.py

```diff
@@ -109,6 +129,15 @@ def __str__(self):
                 pk=self.pk,
             ))
 
+    @property
+    def config(self):
+        last_build = (
+            self.builds.filter(type='html', state='finished')
```

Comment:
Do we still use the `type` field? It seems old and unnecessary at this point.

Reply:
Yup, I was thinking the same.